src/backend/distributed/executor/transmit.c

```diff
@@ -310,113 +312,3 @@ ReceiveCopyData(StringInfo copyData)
 
 	return copyDone;
 }
-
-
-/* Is the passed in statement a transmit statement? */
-bool
-IsTransmitStmt(Node *parsetree)
-{
-	if (IsA(parsetree, CopyStmt))
-	{
-		CopyStmt *copyStatement = (CopyStmt *) parsetree;
-
-		/* Extract options from the statement node tree */
-		DefElem *defel = NULL;
-		foreach_ptr(defel, copyStatement->options)
-		{
-			if (strncmp(defel->defname, "format", NAMEDATALEN) == 0 &&
-				strncmp(defGetString(defel), "transmit", NAMEDATALEN) == 0)
-			{
-				return true;
-			}
-		}
-	}
-
-	return false;
-}
-
-
-/*
- * TransmitStatementUser extracts the user attribute from a
- * COPY ... (format 'transmit', user '...') statement.
- */
-char *
-TransmitStatementUser(CopyStmt *copyStatement)
```

Comment:
I guess we still use the word `transmit`? If we decide to change (to `copy_utils.c` or such), maybe we rename `FileOpenForTransmit` function in this file as well?